code/gradle.properties

```diff
@@ -24,6 +24,4 @@ mavenCoreVersion=2.0.1
 mavenLifecycleVersion=2.0.0
 mavenIdentityVersion=2.0.0
 mavenSignalVersion=2.0.0
```

Comment:
Let's update to latest version `2.0.1` for Identity, Lifecycle and Signal extensions